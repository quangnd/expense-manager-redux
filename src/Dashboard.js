import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";

const Dashboard = () => (
  <div>
    This is from my dashboard component!
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default Dashboard;
