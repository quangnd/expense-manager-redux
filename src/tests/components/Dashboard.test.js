import React from "react";
import { shallow } from "enzyme";
import Dashboard from "../../Dashboard";

test("should render ExpenseListItem correctly", () => {
  const wrapper = shallow(<Dashboard />);
  expect(wrapper).toMatchSnapshot();
});
