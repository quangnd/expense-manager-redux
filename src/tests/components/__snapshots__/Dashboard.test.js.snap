// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly 1`] = `
<div>
  This is from my dashboard component!
  <ConnectFunction />
  <ConnectFunction />
</div>
`;
